src/components/ZenModeAlertDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Dialog } from "./Dialog";
+
+import { t } from "../i18n";
+
+import "./ZenModeAlertDialog.scss";
+
+type Props = {
+  onClose: () => void;
+  onContinue: () => void;
+  onExit: () => void;
+};
+const ZenModeAlertDialog = (props: Props) => {
+  const { onClose, onContinue, onExit } = props;
+  return (
+    <Dialog
+      onCloseRequest={onClose}
+      title={t("zenModeAlertDialog.title")}
+      maxWidth={500}
+      className="zen-mode-alert"
+    >
+      <div>
+        {t("zenModeAlertDialog.content")}
+        <div className="zen-mode-alert__footer">
```

Comment:
Wow thanks for the effort of putting buttons as well! They look great.

I wonder if they are too much, maybe just some text that says "You are in zen mode, type ALT+Z to exit.".

@dwelle will probably have a better opinion.

Reply:
@kbariotis I think if we have buttons in the dialog in the UI becomes more user friendly so the user doesn't have to take the step of pressing the keys Alt+Z. Does that help ?